### PR TITLE
Add additional project tag and component tags

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/16673
https://github.com/cityofaustin/atd-data-tech/issues/16811

I forgot the Map Bikeways 2025 tag in this PR https://github.com/cityofaustin/atd-moped/pull/1313.

edited to add: And Nathan forgot two tags off the list, adding them here too. 

Also when testing this PR, I noticed that the tags were only being alphabetically sorted by the Type, which meant that 2024 was showing up ahead of 2019. I am further sorting them by name. 

## Testing

run locally, check for the missing tag in the component tag drop down, and for the missing tag in the project tags

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [ ] ~~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~~

### CODE REVIEW

**Comment by amenity** [comment]
No pressure, but in case you want to smush this one in here before people review: https://github.com/cityofaustin/atd-data-tech/issues/16811

**Review by mddilley — APPROVED** [review]
Ran locally, and I see the new component tag! The updated order looks great too. 🙏 🚢

**Comment by mddilley** [comment]
@chiaberry just saw Amenity's comment above. Feel free to re-request if that goes in here. 🙏

File: moped-database/migrations/1713206648608_add_2025_component_tag/up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO public.moped_component_tags ("type", name, slug) VALUES
+('MAP Bikeways', '2025', 'map_bikeways_2025'),
+('MAP Ped Crossing', '', 'map_ped_crossing');
```
Comment (roseeichelmann) [inline moped-database/migrations/1713206648608_add_2025_component_tag/up.sql:3]: i dont think im seeing this one show up? i do see the map bikeways one though.

**Review by roseeichelmann — APPROVED** [review]
test steps checked out, up and down migrations worked perf :rocket:

**Review by mddilley — APPROVED** [review]
Tested the new tag and the migration again - all looks 💯 🚀 🚢 !!!

**Review by tillyw — APPROVED** [review]
tested locally, up and down migrations work smoothly and the new order looks really nice and clean! ✨

**Review by amenity — APPROVED** [review]
👍🏻 🚲 🏷️ 🚢  thanks!

**Pull request merged**